src/app/public/plugin-resources/code-examples/checkbox/basic/checkbox-demo.component.html

```diff
@@ -0,0 +1,42 @@
+<form
```

Comment:
When I open this in Stackblitz and select the checkboxes, the color changes to indicate that they are selected but they don't have the checkmarks inside the checkboxes. Is that the expected behavior?

Reply:
Known issue - see answer below.